build/ci/release/Jenkinsfile

```diff
@@ -63,3 +72,10 @@ pipeline {
     }
 
 }
+
+def releaseImageNotExist() {
+    return sh (
+        script: "docker pull $LATEST_RELEASED_IMG &>/dev/null; echo \$?",
+        returnStatus: true
```

Comment:
Not sure we need `; echo \$?` given we set `returnStatus: true`.

Reply:
Fixed